docs/config.md

```diff
@@ -221,5 +221,12 @@ improvement, as well as a reduction in memory usage.
 - `DisableNatPortMap`
 Disable NAT discovery.
 
+- `DisableRelay`
+Disables the p2p-circuit relay transport.
```

Comment:
Setting to true will disable the relay transport entirely.

The `DislableRelay` controls option 1 and 3, so by default we listen for relay connections and we can connect to peers through relay.
In order to act as a hop relay, you need to `EnableRelayHop`, in addition to having the relay transport enabled.

I don't think it makes sense to have more fine-grained independent configuration, as we can't really separate listening from dial (and do we really want to do that?) and we already have an option to control hop relaying.